packages/block-editor/src/utils/format-font-weight.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Formats font weights to human readable names.
+ *
+ * @param {string} fontWeight font weight string
+ * @return {Object} new object with formatted font weight
+ */
+export function formatFontWeight( fontWeight ) {
+	if ( ! fontWeight ) {
+		return {};
+	}
+
+	if ( typeof fontWeight === 'object' ) {
+		return fontWeight;
+	}
+
+	let name;
+
+	switch ( fontWeight ) {
+		case 'normal':
+			name = _x( 'Regular', 'font weight' );
+			break;
+		case 'bold':
```

Comment:
Is it necessary to consider colors other than `normal` and `bold`, such as `thin` and `black`?

Then, I think the case statements can be put together.

```javascript
		case 'normal':
		case '400':
			name = _x( 'Regular', 'font weight' );
			break;
```

Reply:
I didn't realise both `400` and `normal`, and `700` and `bold` could be combined, thanks! I've done that here: https://github.com/WordPress/gutenberg/pull/61915/commits/672ae3afdf3be1be1369220ce8c65b4e95668104.

Does this address your original comment? Or do you think we should remove `thin`, `extra light`, `light`, `medium`, `semi bold`, `extra bold`, and `black`? I added these as this follows the [CSS spec for `font-weight`](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight), and it's closer to how we're currently showing these values.

Reply:
> Or do you think we should remove thin, extra light, light, medium, semi bold, extra bold, and black

The question was not for me, but I think those are worth keeping; fonts do offer that level of customization.

Reply:
`thin` and `black` aren't valid CSS values for font-weight, but only labels that are mapped to numerical weights by convention. So to me this now looks good to go: check for the numerical values 100 - 900 or the text values `normal`/`bold`, and map those to a common label, when possible.